### DISCUSSION TITLE
Who's Tried the Smart Connections for Obsidian GPT? 🚀 Let's Share Insights!

### DISCUSSION THREAD

**Discussion by humouribicon** [brianpetro/obsidian-smart-connections#467] [Q&A]
Hey Obsidian Enthusiasts! 🌟
-- 
EDIT : I finally became a supporter (see my update in below comments)
--
I'm diving straight into it - has anyone here taken the Smart Connections for Obsidian GPT for a spin? This tool is like a personal assistant for our Obsidian vaults, making it a breeze to sift through folders, find specific notes, and get exactly what we need from our sea of Markdown files.

I'm super curious about your experiences! Did it change your note-taking game? Did you find it easier to manage your ideas or projects? Any cool tips or tricks you've discovered while using it? And here's the kicker - how does using this GPT stack up against using the plugin directly in Obsidian? If there's a difference, I'm all ears to understand the potential benefits!

Also, there's a bit about needing to link up with a Smart Connections app account. I get it, it's all about keeping our notes in sync and making sure we can access them across devices. But I'd love to hear your take on it - how's that working out for you?

Let's turn this thread into a treasure trove of user experiences and wisdom! 📚💡 Whether you're a seasoned user or just thinking about giving it a try, your insights could be super helpful to the community.

Looking forward to your stories and tips!
Cheers.

**Comment by visualcurrent** [comment]


**Reply by brianpetro** [reply]
@visualcurrent 

> bait and switch

I know this is the internet, but false accusations like this about someone who is doing their best to provide free open-source software that enables individuals to succeed is seriously offensive.

It's true that the Obsidian plugin does NOT require a supporter key.

Unfortunately, it wouldn't be possible for me to subsidize the >100K people that have downloaded Smart Connections to use the ChatGPT integrations which incur an additional cost to me for every user.

So, if you want to try the experiments, like the ChatGPT integrations, which require a significant cost to myself through development and hosting of cloud servers, then yes, you need a supporter key. 

In the near future, after supporters have helped me by beta testing the early release of Smart Connect, the new advanced ChatGPT integration:

1) there will be a clear product offering.

2) a significant portion of the integration will be made open source. 

3) for people that have more time than money, I will provide instructions for ways to try/utilize the software WITHOUT paying me anything (by either running your own backend or connecting to a third-party).

But for now, I'm still working on building something important. And to do that, I require real support.

🌴

**Reply by visualcurrent** [reply]
That's awesome that you are trying to provide free open-source software that enables individuals to succeed. I'm truly sorry that I offended. Your passionate response alerts me that I may have misinterpreted what's going on.

But it does seem like a switch is what's happening when a new user like me hits a wall like that. I may have approached your creation incorrectly, but I'm just reporting my experience. If you're not doing a bait and switch, and my response is offensive, maybe a little more clarity is needed in the process for new users like me (what I'm calling transparency above).

> "Unfortunately, it wouldn't be possible for me to subsidize the >100K people that have downloaded Smart Connections to use the ChatGPT integrations which incur an additional cost to me for every user." 

This info is not clear when a new user like me is diving in and trying stuff out. Or maybe it was there and I just missed it as I was trying to get it working. But I didn't see it so maybe more than just me will have a frustrating entry. I have a Chat-GPT Plus subscription and I assumed that my API would allow me to use it.

It sounds like you're saying the Obsidian plugin does NOT require a supporter key but the plugin does require a supporter key for it to be used. That's misleading also. Again, I may be misinterpreting but that's what it sounds like you're saying. If that's the case, then just make it clear to people that they can install the plugin if they want but they can't use it without the supporter key. It's awesome what you're doing but people need clarity.

This is all new to me and I'm just trying to find ways to use it. I sincerely apologize for my offense

**Reply by visualcurrent** [reply]
I'm happy to delete my comment also. I don't want to taint other people's experience in a negative way. Just let me know if that would be better.

**Reply by visualcurrent** [reply]
I think all you have to do is make some of these points clear. Not to me here. But in your documentation for new users to see.

**Reply by visualcurrent** [reply]
Please feel free to delete my thread. It wouldn't let me do it other than my own comments. I may come back later and start over with a more compassionate attitude and with the intention of giving you the experience of a newcomer. When you're deep in a project, sometimes is useful to get a noob's perspective, especially if you're wanting to build community because that's where is going to be coming from. New users. 

Regards

**Comment by miracletamer** [comment]
Just found it.... :o) - will let you know

**Reply by brianpetro** [reply]
Hey @miracletamer 

The latest software for new ChatGPT integration, and the link to the Smart Connections 2 GPT, can be found in the private community for supporters at this link https://chat.smartconnections.app/rooms/5/@41

If you haven't signed in to the chat before, you can by using your supporter key and the same email you used to receive your supporter key.

Looking forward to hearing your thoughts!

🌴

**Reply by humouribicon** [reply]
🎉🎈 Yeah, great! I’m really looking forward to your feedback. ❤️‍🔥

**Reply by micuentadecasa** [reply]
Hi @brianpetro , how can I be a supporter?

**Comment by SoulBits-Vibe** [comment]
This works so well! I'm using it to teach myself Obsidian, and it's much faster than leaving the note to use outside references. The responses I get from within my own work are incredibly useful. This plugin is bringing me closer to my dream of an all-in-one space.

Great work on this! Over the years, I've used many different apps and tested/played with various new AI tools. This plugin, however, is not just a plaything; it's a keeper. By working alongside me as I learn Obsidian, it has been the only successful attempt I've had towards getting used to using this system.  


my experience over the past couple of weeks - Feb. 2024

### **Comparing Smart Connections:**
Faster than using any web browser interfaced AI. 
Gemini replies fast but getting to it is several clicks away.  And Gemini isa very frustrating experience trying to research. 
Much faster than copilot. Copilot is always soooo slow with reply text.  
It understands the context with less prompt arrangements. I don't have to overthink the query. 
I can get personal answers such as "when did I buy item" or "is the warranty still good on my appliance."  No one else can do that.
Having the screen in sidebar while working inside Obsidian is gold - no losing focus or time tabbing out. 


Someday, I hope the assistant could help manage settings and navigations. And it would be exciting for assistant to edit and create queries or code blocks on demand, essentially making the pages do what I want them to on the fly.  =)

**Comment by micuentadecasa** [comment]
Hi, thanks for creating this superb plugin, I miss some document explaining how to organize the notes and the links between them, I'm going to put below a use case:
- one folder for projects
- one folder for people
- one folder for meeting minutes
and now, if I go to a project folder and ask about the people on this project or about the latest meeting minutes for a project? e.g., "plan the next meeting for Project X based on the latest meetings"

so the question is how we can organize the notes and their relationships to be able to get all the notes related in the prompt ?

**Comment by iankordonis** [comment]
I love this plug-in. I use ChatGPT for the api key. I think its been extremely helpful in building new thoughts off of existing ones. I am wondering however, has anyone had any luck getting smart connections to read embedded pdf's? I'm having trouble myself

**Reply by brianpetro** [reply]
@iankordonis I'm happy to see that you're loving the plugin 😊

PDFs are currently unsupported. Some people have had luck by converting PDFs to markdown, but this process can be difficult/unpredictable 🌴

**Reply by iankordonis** [reply]
What about being able to edit chat history? I can't seem to delete any ones that I don't want

![smart connection s](https://github.com/brianpetro/obsidian-smart-connections/assets/128556686/82b5b677-99a9-41ab-b691-760c051a5ae8)

**Reply by brianpetro** [reply]
`v2.1` will introduce chat history as markdown notes, so managing it will be easy 🌴

**Reply by iankordonis** [reply]
Beautiful!! When is the release date?

**Reply by brianpetro** [reply]
I was aiming to have it in beta testing with supporters by now. So that could happen any day now. I'm working on the "final checklist" for that part today 🌴

**Comment by Evilinside3357** [comment]
Might be a dumb question, But is the cost for the supporter license 100$ minimum? Or is it flat 200$?

**Reply by brianpetro** [reply]
$200 suggested, $100 minimum 🌴

**Comment by humouribicon** [comment]
Hello, I notice there are token limitations on ChatGPT 4, is this also the case with the GPT of Smart Connections?

**Reply by brianpetro** [reply]
Are you referring to https://chat.openai.com/g/g-9Xb1mRJYl-smart-connections-2 ? 

In any case, there are limits on context for all language models like ChatGPT. 

🌴

**Comment by sulth1** [comment]
Hi! The creator of the Copilot plugin released a major update that now allows the user to embed their vault and ask questions about it. How does SmartConnections differ from the new Copilot vault QA function?
Thanks, have a great day!

**Reply by brianpetro** [reply]
Hi @sulth1 
From the beginning, I've made it easy for other plugins to re-use Smart Connections minimal/no dependency embeddings to prevent exposing vaults to unnecessary security vulnerabilities. And I have a very specific vision for the future of this plugin. So I'm not sure what other plugins might be doing. But it sounds like a good question to ask their creators 🌴

**Comment by Kmfernan5** [comment]
It's too confusing I keep getting cut off and being asked to be a paying member can you use it free if so how? In the meantime moving over to Copilot and second brain plugins.

**Reply by christianngarcia** [reply]
You can try it for free, see this video https://youtu.be/TDUWvOc7xJo?si=Buai2Vb7cMAPDDxF 🌼

**Comment by humouribicon** [comment]
Hello brian, I’ve been using smart connection in obsidian since about a year and loving it. 

I wonder what is the advantage to use le gpt instead of the plug in? 

I wonder if I can « modify » it by sending pdf to smart connection like I do when a create my own gpt ? 

If not, do you know how could I combine this two features ? Using chat gpt with my obsidian notes AND  modify the gpt by sending some pdf ?

ps: I Just saw there is top gpt now (smart Connect ans smart connections). What is the différence ?

**Reply by brianpetro** [reply]
Is something like this what you're trying to do with PDFs? https://www.youtube.com/watch?v=909wdjg71fM

There is an older GPT that is still available but will be discontinued soon. This is the primary GPT that uses the new Smart Connect software https://chatgpt.com/g/g-9Xb1mRJYl-smart-connect-obsidian

🌴

**Reply by humouribicon** [reply]
I am a native of France, so maybe I didn't understand everything correctly.

Could you please tell me if I understood it the right way?

1- Do you use only the Smart Connect plugin, or do you also use GPT?

2- If the language model was able to summarize the listed PDFs, it means the language model can read the PDFs stored in Obsidian, right?

3- Is it also possible to do this "remotely" when using GPT Smart Connect in the Chat GPT app?

**Reply by brianpetro** [reply]
1) Smart Connect_ions_ is the Obsidian plugin. Smart Connect is the app that enables ChatGPT integration. Currently, it's best to use both together.

2) Yes, but this is currently limited to the ChatGPT integration ([Smart Connect - Obsidian GPT](https://chatgpt.com/g/g-9Xb1mRJYl-smart-connect-obsidian))

3) When the Smart Connect app is running on your desktop, you can use the Smart Connect GPTs to access the files on your desktop via ChatGPT from any device.

🌴

**Reply by humouribicon** [reply]
Thx ! Great, I'm considering buying Smart Connect, but I didn't understand if the access is permanent... I saw in the settings of the Obsidian plugin that if I became a supporter, my access would be 'for a very limited time'. 
1- Can you tell me more about this? 
2- Is it here where I should pay 100 dollars minimum to buy Smart Connect ? https://buy.stripe.com/9AQ7sWemT48u1LGcN4

**Reply by brianpetro** [reply]
Supporters _do not_ get permanent access to the [Smart Connect Official service](https://smartconnections.app/smart-connect/). Currently, supporters receive access until December 2024. Soon, that will be reduced to three months since you can now pay monthly. Becoming a supporter is for people who sincerely believe they should support the software they use, and is not any sort of quid pro quo agreement for services 🌴

**Reply by humouribicon** [reply]
Hello, I bought it and I am very happy with it. I have a question: now that it is installed on my MacBook and connected to my Obsidian vault, will the ChatGPT Smart Connect only take into account the notes I create from my iPhone once I have restarted Obsidian on my Mac, is that correct?

**Reply by brianpetro** [reply]
Hi @humouribicon 

Smart Connect will access all the notes in your vault on the computer where the app is running.

If you have Obsidian sync, and your Mac is connected, the new notes you create on your iPhone should be available to Smart Connect almost immediately (Obsidian sync works pretty quick).

🌴

**Reply by humouribicon** [reply]
Okay, thank you. That's clearer. So, I have a kind of impression that the old plugin, which required uploading notes, although it required a GPT Plus subscription, had the advantage of synchronizing the notes with Smart Connections without needing to have the Mac turned on.

**Reply by brianpetro** [reply]
Some features for "offline" (when disconnected) usage is something I think will be added to the Smart Connect Official service in the future 🌴

**Reply by humouribicon** [reply]
Omg I juste realized that ! It’ s such a pity. I absolutely cannot let my MacBook open and connected all the time… 😭

**Reply by humouribicon** [reply]
meanwhile, is there a way to use the older gpt that requires to upload my notes ?

**Comment by humouribicon** [comment]
-- 
I FINALLY BECAME A SUPPORTER AND HERE’S WHY
--
> Hey Obsidian Enthusiasts! 🌟
> 
> I'm diving straight into it - has anyone here taken the Smart Connections for Obsidian GPT for a spin? This tool is like a personal assistant for our Obsidian vaults, making it a breeze to sift through folders, find specific notes, and get exactly what we need from our sea of Markdown files.
> 
> I'm super curious about your experiences! Did it change your note-taking game? Did you find it easier to manage your ideas or projects? Any cool tips or tricks you've discovered while using it? And here's the kicker - how does using this GPT stack up against using the plugin directly in Obsidian? If there's a difference, I'm all ears to understand the potential benefits!
> 
> Also, there's a bit about needing to link up with a Smart Connections app account. I get it, it's all about keeping our notes in sync and making sure we can access them across devices. But I'd love to hear your take on it - how's that working out for you?
> 
> Let's turn this thread into a treasure trove of user experiences and wisdom! 📚💡 Whether you're a seasoned user or just thinking about giving it a try, your insights could be super helpful to the community.
> 
> Looking forward to your stories and tips! Cheers.

I'm replying to myself after testing GPT by becoming a supporter: IT'S ABSOLUTELY WORTH IT! I am thrilled with this tool, and I can never thank the creator or creators of this GPT enough. I am not very well informed about how many people are behind and responsible for this project, and since I am French, I have trouble understanding all the exchanges on GitHub. In any case, I also thank all the supporters who have enabled the evolution of this project, and really, I am delighted, delighted, delighted. I invite you all to use it. And for me, the main reason is that it allows me to exponentially exploit all my notes in Obsidian and truly leverage my second brain. 🌎🌈🤘🥰
